src/app/shared/sidebar/sidebar.component.ts

```diff
@@ -46,13 +46,17 @@ export class SidebarComponent implements OnInit {
     this.loading = this.store.pipe(select(selectResourceIsLoading('individual')));
   }
 
-  public toggleSectionCollapse(sectionIndex: number): void {
-    this.store.dispatch(new fromSidebar.ToggleCollapsibleSectionAction({ sectionIndex }));
+  public onSelectSection(section: SidebarSection, sectionIndex: number): void {
+    if (section.useDialog) {
+      console.log('dispatch open dialog action');
```

Comment:
Is this log for debugging?
Do you intend for this console log to be printed during normal use?

Reply:
This is just a place holder log. It will be removed when dispatching the action to open a yet to be implemented multi-select modal dialog.

Reply:
Removed. Thanks.